spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.describe 'User index integration tests', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+  before(:all) do
+    @users = User.all
+  end
+  it 'shows all the properties (Name, photo, number of posts) of each user' do
+    visit users_path
+    @users.each do |user|
+      expect(page).to have_text(user.Name)
+      expect(page).to have_content('Number of posts')
+
+      expect(page).to have_text(user.PostsCounter)
+    end
```

Comment:
- [ ] As the previous reviewer suggested that you should have a test to check if the user has a photo or not :+1: Either you can add an expect in this block or you can make a new it block to check photo of a user :+1:

Reply:
resolve problem